core/src/test/scala/integration/kafka/server/KRaftClusterTest.scala

```diff
@@ -1476,7 +1484,8 @@ class KRaftClusterTest {
         val parentDir = log.parentDir
         val targetParentDir = broker0.config.logDirs.filter(_ != parentDir).head
         val targetDirFile = new File(targetParentDir, log.dir.getName)
-        FileUtils.copyDirectory(log.dir, targetDirFile)
+        targetDirFile.mkdir()
+        copyDirectory(log.dir.toString(), targetDirFile.toString())
```

Comment:
Is this unrelated?

Reply:
This test used commons-io, which is being removed. It's generally redundant with the new Java stuff, but was a ZK dependency.